### PR TITLE
read json values to properties

### PR DESCRIPTION
**Pull request by azhamn**

### CODE REVIEW

File: com.wso2telco.dep.hub.paymentapi/.project [hunk 1]
```diff
@@ -5,8 +5,14 @@
...
 	</projects>
 	<buildSpec>
+		<buildCommand>
+			<name>org.eclipse.m2e.core.maven2Builder</name>
+			<arguments>
+			</arguments>
+		</buildCommand>
 	</buildSpec>
 	<natures>
+		<nature>org.eclipse.m2e.core.maven2Nature</nature>
```
Comment (rajithsiriwardana) [inline com.wso2telco.dep.hub.paymentapi/.project:15]: you don't need these lines.

File: com.wso2telco.dep.hub.paymentapi/com.wso2telco.dep.hub.paymentapi.synapse/.project [hunk 2]
```diff
@@ -5,8 +5,14 @@
 	<projects>
 	</projects>
 	<buildSpec>
+		<buildCommand>
+			<name>org.eclipse.m2e.core.maven2Builder</name>
+			<arguments>
+			</arguments>
+		</buildCommand>
```
Comment (rajithsiriwardana) [inline com.wso2telco.dep.hub.paymentapi/com.wso2telco.dep.hub.paymentapi.synapse/.project:12]: same goes here. check what are the files you are adding for the commit.

File: com.wso2telco.dep.hub.paymentapi/com.wso2telco.dep.hub.paymentapi.synapse/src/main/synapse-config/api/paymentAPI.xml [hunk 3]
```diff
@@ -1,7 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <api context="/payment" name="paymentAPI" xmlns="http://ws.apache.org/ns/synapse">
   <resource methods="POST" protocol="http" uri-template="/{endUserId}/transactions/amount">
-    <inSequence/>
+    <inSequence>
+      <sequence  key="com.wso2telco.dep.hub.paymentapi.handle.charge.Sequence"/>
+    </inSequence>
     <outSequence/>
     <faultSequence/>
```
Comment (rajithsiriwardana) [inline com.wso2telco.dep.hub.paymentapi/com.wso2telco.dep.hub.paymentapi.synapse/src/main/synapse-config/api/paymentAPI.xml:8]: add a fault sequence

Comment (shazni) [reply com.wso2telco.dep.hub.paymentapi/com.wso2telco.dep.hub.paymentapi.synapse/src/main/synapse-config/api/paymentAPI.xml:8]: Rajith. This is a work in progress. It'll be added down the line,

Comment (shazni) [reply com.wso2telco.dep.hub.paymentapi/com.wso2telco.dep.hub.paymentapi.synapse/src/main/synapse-config/api/paymentAPI.xml:8]: Rajith. This is work in progress. Those will be added as it goes along.

**Pull request merged**